### PR TITLE
gz-harmonic: new formula

### PR DESCRIPTION
**Pull request by azeey**
See https://github.com/gazebo-tooling/release-tools/issues/976

### CODE REVIEW

File: Formula/gz-harmonic.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+class GzGarden < Formula
+  include Language::Python::Virtualenv
+
+  desc "Collection of gazebo simulation software"
+  homepage "https://github.com/gazebosim/gz-garden"
```
Comment (iche033) [inline Formula/gz-harmonic.rb:5]:
```suggestion
  homepage "https://github.com/gazebosim/gz-harmonic"
```

File: Formula/gz-harmonic.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  depends_on "cmake" => :build
+  depends_on "python@3.11" => [:build, :test]
+
+  depends_on "gz-cmake3"
+  depends_on "gz-common5"
+  depends_on "gz-fuel-tools8"
+  depends_on "gz-gui7"
+  depends_on "gz-launch7"
+  depends_on "gz-math7"
+  depends_on "gz-msgs9"
```
Comment (iche033) [inline Formula/gz-harmonic.rb:21]: `gz-msgs10`

File: Formula/gz-harmonic.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  head "https://github.com/gazebosim/gz-garden.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.11" => [:build, :test]
+
+  depends_on "gz-cmake3"
+  depends_on "gz-common5"
+  depends_on "gz-fuel-tools8"
+  depends_on "gz-gui7"
```
Comment (iche033) [inline Formula/gz-harmonic.rb:18]: `gz-gui8`

File: Formula/gz-harmonic.rb [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  depends_on "gz-launch7"
+  depends_on "gz-math7"
+  depends_on "gz-msgs9"
+  depends_on "gz-physics7"
+  depends_on "gz-plugin2"
+  depends_on "gz-rendering7"
```
Comment (iche033) [inline Formula/gz-harmonic.rb:24]: `gz-rendering8`

File: Formula/gz-harmonic.rb [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  depends_on "gz-sensors8"
+  depends_on "gz-sim8"
+  depends_on "gz-tools2"
+  depends_on "gz-transport12"
```
Comment (iche033) [inline Formula/gz-harmonic.rb:28]: `gz-transport13`

File: Formula/gz-harmonic.rb [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  license "Apache-2.0"
+
+  head "https://github.com/gazebosim/gz-garden.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.11" => [:build, :test]
+
+  depends_on "gz-cmake3"
+  depends_on "gz-common5"
+  depends_on "gz-fuel-tools8"
```
Comment (iche033) [inline Formula/gz-harmonic.rb:17]: `gz-fuel-tools9`

File: Formula/gz-harmonic.rb [hunk 7]
```diff
@@ -0,0 +1,55 @@
+class GzGarden < Formula
+  include Language::Python::Virtualenv
+
+  desc "Collection of gazebo simulation software"
+  homepage "https://github.com/gazebosim/gz-garden"
+  url "https://github.com/gazebosim/gz-harmonic.git", branch: "main"
+  version "0.999.999~0~20230728"
+  license "Apache-2.0"
+
+  head "https://github.com/gazebosim/gz-garden.git", branch: "main"
```
Comment (iche033) [inline Formula/gz-harmonic.rb:10]:
```suggestion
  head "https://github.com/gazebosim/gz-harmonic.git", branch: "main"
```

**Comment by azeey** [comment]
I've addressed all the feedback. PTAL.

File: Formula/gz-harmonic.rb [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+  depends_on "gz-msgs10"
+  depends_on "gz-physics7"
+  depends_on "gz-plugin2"
+  depends_on "gz-rendering8"
+  depends_on "gz-sensors8"
+  depends_on "gz-sim8"
+  depends_on "gz-tools2"
+  depends_on "gz-transport13"
+  depends_on "gz-utils2"
+  depends_on macos: :mojave # c++17
```
Comment (scpeters) [inline Formula/gz-harmonic.rb:30]: nit: this line can be removed

Comment (scpeters) [reply Formula/gz-harmonic.rb]: https://github.com/osrf/homebrew-simulation/pull/2326/commits/7c5e58ce991012f16f92b61c43890c797c8a8323

**Review by scpeters — APPROVED** [review]

**Pull request merged**